src/db.rs

```diff
@@ -321,6 +328,14 @@ impl Group {
             }
         }
     }
+
+    /// Get a timestamp field by name
+    pub fn get_time(&self, key: &str) -> Option<&chrono::NaiveDateTime> {
+        match self.times.get(key) {
+            Some(ref t) => Some(t),
+            _ => None,
+        }
+    }
```

Comment:
similar to the `get_username` etc. methods, a `get_expiry_time` method would be a nice to have.

Reply:
I will add this method.